config.toml

```diff
@@ -9,9 +9,9 @@ style = "paraiso-dark"
 
 [params]
 font_awesome_version = "5.12.0"
-description = "A project for inclusive language and terminology in open source and technology"
+description = "Helping projects and companies make consistent, responsible choices to remove harmful language"
```

Comment:
Can this be phrased in the positive?

Reply:
@xmulligan any thoughts?

I don't think this is a blocker for merge, but it'd be good to revise.

Reply:
Absolutely not a blocker.  Please do merge.